### PR TITLE
Demonstrate scrolling dialogs

### PR DESCRIPTION
**Pull request by yschimke**
#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

**Comment by yschimke** [comment]
cc @ssancho14

**Comment by yschimke** [comment]
Landing for an offline comparison.

File: sample/src/test/kotlin/com/google/android/horologist/screensizes/DialogTest.kt [hunk 1]
```diff
@@ -70,4 +80,54 @@ class DialogTest(device: Device) : ScreenSizeTest(
...
+        lateinit var columnState: ScalingLazyColumnState
+
+        runTest(testFn = {
+            screenshotTestRule.interact {
+                runBlocking {
+                    columnState.state.scrollToItem(999, 0)
+                }
+            }
+
+            screenshotTestRule.takeScreenshot()
```
Comment (luizgrp) [inline sample/src/test/kotlin/com/google/android/horologist/screensizes/DialogTest.kt:95]: should this be called from `ScreenSizeTest.runTest` or is the intention to let each test to decide when to take a screenshot?

File: sample/src/test/kotlin/com/google/android/horologist/screensizes/DialogTest.kt [hunk 2]
```diff
@@ -70,4 +80,54 @@ class DialogTest(device: Device) : ScreenSizeTest(
             }
         }
     }
+
+    @Test
+    fun longDialogScreen1() {
```
Comment (luizgrp) [inline sample/src/test/kotlin/com/google/android/horologist/screensizes/DialogTest.kt:85]: nit: drop 1 from name

**Review by luizgrp — COMMENTED** [review]
LGTM

File: sample/src/test/kotlin/com/google/android/horologist/screensizes/DialogTest.kt (hunk 1 above)
Comment (yschimke) [reply sample/src/test/kotlin/com/google/android/horologist/screensizes/DialogTest.kt:95]: Not sure I wanted the minimal way to let me get multiple screenshots.  I should improve the API.  Will follow up.

**Pull request merged**